OneCommunityGlobal/HighestGoodNetworkApp
# Supriya -Add-Projects-dropdown-in-bmdashboard-with-404-NotFound-page

# Description
1. Add the Projects dropdown only after selecting the project and enters into its dashboard
2. Show the 404 page if any page is not designed yet or a new link is entered. 

<img width="518" alt="image" src="https://github.com/user-attachments/assets/63beb123-5b76-4153-af54-7f74ba95599e">


## Related PRS (if any):
You can checkout with same PR in backend 
…

## Main changes explained:
1. Added a dropdown for Projects in the bmdashboard header (after selecting a particular project)
2. Added the specific pages to the specific items in the dropdown 
4. Added a 404 - Not Found Page for any links which are not existed will show this page
5. Added the 404 page in dark mode too
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. go to bmdashboard -> Select a Project -> Go to Project Dashboard -> Projects(in Header) -> check the dropdown Pages
    http://localhost:3000/bmdashboard/login
6. Verify each and every page in dropdown list
7. If no page is created then it leads to 404 page (Not found) (dark mode too)
8. Verify with any random links to check 404 page entire App.

## Screenshots or videos of changes:
[

https://github.com/user-attachments/assets/2afeb8d1-04c9-46bf-82d5-c72537f63abf

](url)

## Note:
Pages in the dropdown list are not in the dark mode yet
